src/playlist-maker.js

```diff
@@ -8,6 +8,10 @@ var indexInSources = function(arr, src) {
   var j = 0;
   var item;
   var source;
+  //If there is an rtmp source in the playlist, src comes out as undefined and indexOf incorrectly gives out the value as 0
+  //hence we make sure to hit the condition that if src is undefined just return -1 in the beginning itself
+  if(src === undefined)
```

Comment:
Our style conventions in this project have different whitespace and require `{}`s around the condition:
```js
if (src === undefined) {
  // ...
}
```